spec/spec_helper.rb

```diff
@@ -116,6 +116,15 @@
   config.add_setting :reload_page_wait_time, default: 60
 end
 
+chromedriver_from_path = File.which("chromedriver")
```

Comment:
This code looks quite useful, I wonder if we could move it into `govuk_test` somehow and then use it here?

Reply:
I've already got a Pull Request open for `govuk_test` to make a similar change there https://github.com/alphagov/govuk_test/pull/7